ftp/1.8/modules/ROOT/pages/ftp-documentation.adoc

```diff
@@ -701,7 +701,7 @@ Will load all components in the pool on startup. |  |
 |===
 
 [[HttpsTunnelProxy]]
-=== HTTP Tunnel Proxy
```

Comment:
should we fix in previous versions too?

Reply:
Thanks for the review. Added the patch for previous versons